components/dash-core-components/package.json

```diff
@@ -15,7 +15,7 @@
     "private::format.black": "black dash_core_components_base/ tests/ setup.py",
     "private::format.eslint": "eslint src --fix",
     "private::format.prettier": "prettier --config .prettierrc --write src/**/*.js",
-    "private::lint.black": "node -e \"if ((process.env.PYVERSION || 'python310') !== 'python36'){process.exit(1)} \" || black --check dash_core_components_base/ tests/ setup.py",
+    "private::lint.black": "node -e \"if ((process.env.PYVERSION || 'python312') !== 'python38'){process.exit(1)} \" || black --check dash_core_components_base/ tests/ setup.py",
```

Comment:
I suppose we can leave this as you have it for now, but way better would be to pick a version of `black` that supports all the same Python versions we do and always run it. The only reason we didn't do that from the start is that when we first introduced `black` we still supported Py2.7 and `black` was always Py3-only.

Reply:
I'll do this in a follow up PR